app/models/spot.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: spots
+#
+#  id         :integer          not null, primary key
+#  title      :string           not null
+#  latitude   :decimal(, )      not null
+#  longitude  :decimal(, )      not null
+#  radius     :decimal(, )      not null
+#  user_id    :integer
+#  topic_id   :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_spots_on_topic_id  (topic_id)
+#  index_spots_on_user_id   (user_id)
+#
+
+class Spot < ApplicationRecord
+  validates :title, :latitude, :longitude, :radius, presence: true
+  belongs_to :user
```

Comment:
`belongs_to` should be before validations, you can check the order in our style guides https://github.com/rootstrap/tech-guides/blob/master/ruby/rails.md#macro-style-methods